packages/serverless/package.json

```diff
@@ -0,0 +1,90 @@
+{
+  "name": "@sentry/serverless",
+  "version": "1.0.1",
+  "description": "Offical Sentry SDK for Node.js",
+  "repository": "git://github.com/getsentry/sentry-javascript.git",
+  "homepage": "https://github.com/getsentry/sentry-javascript/tree/master/packages/node",
+  "author": "Sentry",
+  "license": "BSD-3-Clause",
+  "engines": {
+    "node": ">=6"
+  },
+  "main": "dist/index.js",
+  "module": "esm/index.js",
+  "types": "dist/index.d.ts",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@sentry/apm": "5.19.1",
+    "@sentry/core": "5.19.1",
+    "@sentry/hub": "5.19.1",
+    "@sentry/node": "^5.20.1",
```

Comment:
Should pin an exact version here. Please update all version numbers to current latest release.

Reply:
Current latest release version has been updated